src/Twig/FacileMongoDbBundleExtension.php

```diff
@@ -27,13 +27,13 @@ class FacileMongoDbBundleExtension extends AbstractExtension
     ];
 
     /**
-     * @return \Twig_SimpleFunction[]
+     * @return TwigFunction[]
      */
-    public function getFunctions()
+    public function getFunctions(): array
     {
         return [
-            new TwigFunction('dataLabelTranslate', [$this, 'queryDataTranslate']),
-            new TwigFunction('isQueryExplainable', [$this, 'isQueryExplainable']),
+            new TwigFunction('dataLabelTranslate', $this->queryDataTranslate(...)),
+            new TwigFunction('isQueryExplainable', $this->isQueryExplainable(...)),
```

Comment:
is this a callable? I've never seen this syntax

Reply:
Yes! First class callable syntax, new since 8.1 😄 Rector applied it here.

https://www.php.net/manual/en/functions.first_class_callable_syntax.php